fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/executable/StringArithmetic.java

```diff
@@ -776,33 +781,15 @@ public static Expression strcmp(StringLikeLiteral first, StringLikeLiteral secon
      */
     @ExecFunction(name = "strleft")
```

Comment:
if strLeft is alias for left, we'd better remove strleft function from fe and add left a alias `strleft` in `fe/fe-core/src/main/java/org/apache/doris/catalog/BuiltinScalarFunctions.java`